src/components/Input/index.js

```diff
@@ -164,9 +175,32 @@ class Input extends Component {
           }}
           rows={1}
         />
+        <div
+          style={{
+            display: 'flex',
+          }}
+        >
+          <button
+            onClick={this.sendMessage}
+            disabled={!value}
+            style={{
+              paddingLeft: 40,
+            }}
+          >
+            <svg
+              style={{
+                width: 40,
+                fill: value ? preferences.accentColor : preferences.botMessageColor,
```

Comment:
Because the button `Send` depends on the preferences, the button doesn't have to be shown unless it is specified in the preference. The style of the input should not changed too if not button appears.

Reply:
That would be another case isn't it? 🤔?
Here I'm handling the disable state triggered by an empty input :).